src/app/shared/http-services/get-group-user-descendants.service.ts

```diff
@@ -0,0 +1,66 @@
+import { HttpClient, HttpParams } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { appConfig } from 'src/app/shared/helpers/config';
+
+interface RawUserDescendant {
+  id: string,
+  name: string,
+  parents: {
+    id: string,
+    name: string,
+  }[],
+  user: {
+    login: string,
+    first_name: string|null,
+    last_name: string|null,
+    grade: number,
+  },
+}
+
+export interface UserDescendant {
+  id: string,
+  name: string,
+  parents: {
+    id: string,
+    name: string,
+  }[],
+  user: {
+    login: string,
+    firstName: string|null,
+    lastName: string|null,
+    grade: number,
+  },
+}
+
+@Injectable({
+  providedIn: 'root'
+})
+export class GetGroupUserDescendantsService {
```

Comment:
Maybe you should group all the descendant services in a single file. No need to have one service per file.

Reply:
On this branch I only have the `GetGroupUserDescendantsService` (the `GetGroupTeamDescendantsService` service was merged on master after I created this branch) so I will do this after the reviews, when I will rebase this branch on master.